frontend/dockerfile/docs/reference.md

```diff
@@ -1617,6 +1618,37 @@ to keep the layer count in the resulting image as low as possible. Therefore,
 with the `--parents` flag, the Buildkit is capable of packing multiple
 `COPY` instructions together, keeping the directory structure intact.
 
+### COPY --exclude
+
+> **Note**
+>
+> Available in [`docker/dockerfile-upstream:master-labs`](#syntax).
+> Will be included in `docker/dockerfile:1.7-labs`.
+
+```dockerfile
+COPY [--exclude=<path> ...] <src> ... <dest>
+```
+
+The `--exclude` flag lets you specify a path expression for files to be excluded.
+
+The path expression follows the same format as `<src>`,
+supporting wildcards and matching using Go's
+[filepath.Match](https://golang.org/pkg/path/filepath#Match) rules.
+For example, to add all files starting with "hom", excluding files with a `.txt` extension:
+
+```dockerfile
+COPY --exclude=*.txt --exclude=*.md hom* /mydir/
```

Comment:
should not exclude `*.md` in this example looking at description above.

```suggestion
COPY --exclude=*.txt hom* /mydir/
```

Reply:
🤦🏻